setup-frontend.js

```diff
@@ -0,0 +1,8 @@
+const FRONTEND_ARGS = ["REACT_APP_CODALAB_RECAPTCHA_SITE_KEY"];
```

Comment:
Instead of hardcoding this, let's just iterate through process.env and pick out the environment variables that start with REACT_APP_ / CODALAB_.

Reply:
Updated.